Services/Maps/js/ServiceOpenLayers.js

```diff
@@ -1,434 +1,387 @@
-		var userPos = new OpenLayers.LonLat(longitude, latitude);
-		return this.toOSMProjection(userPos);
-	};
-
-	// Transforms location from OSM to user projection
-	this.toUserProjection = function (osm) {
-		var _osm = osm.clone();
-		_osm.transform(this.osmProjection, this.userProjection);
-		return _osm;
-	};
-
-	// Transforms location from user projection to OSM projection
-	this.toOSMProjection = function (user) {
-		var _user = user.clone();
-		_user.transform(this.userProjection, this.osmProjection);
-		return _user;
-	};
-
-	// Image for markers...
-	this.getMarkerImage = function() { return new OpenLayers.Icon(
-		"./Services/Maps/images/mm_20_blue.png",
-		new OpenLayers.Size(12, 20),
-		new OpenLayers.Pixel(-6,-20)); };
-
-	// ... and the shadow for it.
-	this.getMarkerShadow = function() { return new OpenLayers.Icon(
-		"./Services/Maps/images/mm_20_shadow.png",
-		new OpenLayers.Size(22, 20),
-		new OpenLayers.Pixel(-7,-20)); };
-
-	// OpenLayers seems to define Icons in a way that only allows
-	// the icon to be placed in one map at one location. Thats
-	// why it is getMarkerImage/Shadow and not a plain icon.
-
-
-
-	// Create a marker on map at latitude, longitude.
-	// Returns an object for marker that has method
-	// setPosition.
-	this.createMarker = function(id, latitude, longitude)
-	{
-		var map = this.maps[id];
-
-		var marker = function (map, latitude, longitude,  module) {
-			var pos = module.OSMPosFromLonLat(longitude, latitude);
-
-			this.marker = new OpenLayers.Marker( pos, module.getMarkerImage() );
-			this.shadow = new OpenLayers.Marker( pos, module.getMarkerShadow() );
-
-			map.__markerLayer__.addMarker(this.shadow);
-			map.__markerLayer__.addMarker(this.marker);
-
-			this.map = map;
-
-			this.pos = pos;
-
-			this.setPosition = (function (self) { return function (loc) {
-				self.map.__markerLayer__.removeMarker(self.marker);
-				self.map.__markerLayer__.removeMarker(self.shadow);
-
-				self.marker.erase();
-				self.shadow.erase();
-
-				self.marker = new OpenLayers.Marker( loc, module.getMarkerImage() );
-				self.shadow = new OpenLayers.Marker( loc, module.getMarkerShadow() );
-
-				self.map.__markerLayer__.addMarker(self.shadow);
-				self.map.__markerLayer__.addMarker(self.marker);
-
-				self.pos = loc;
-			};})(this);
-		};
-
-		return new marker(map, latitude, longitude, this);
-	};
-
-	// Sets the central position (and central marker if used) to the given position.
-	///Sets inputs accordingly.
-	this.jumpTo = function(id, pos)
-	{
-		var map = this.maps[id];
-		map.panTo(pos);
-
-		var userPos = this.toUserProjection(pos);
-
-		jQuery("#" + id + "_address").val("");
-		jQuery("#" + id + "_lat").val(userPos.lat);
-		jQuery("#" + id + "_lng").val(userPos.lon);
-
-		if (map.__replace_marker__) {
-			this.setCentralMarker(id, pos);
+/**
+ * Holds all functionality to work with openlayer maps.
+ *
+ * @author Daniel Weise <daniel.weise@concepts-and-training.de>
+ */
+ServiceOpenLayers = {
```

Comment:
Lets move this complete initialisation into an anonymus function that is called verbatim, so we don't pollute the global namespace.

Reply:
add an var before the object name, so i think it doesn´t pollute the global namespace anymore